docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -100,7 +141,7 @@ Options for the kube-dns `kube-system:kube-dns` ConfigMap:
 | `stubDomains` (optional) | A JSON map using a DNS suffix key (e.g. “acme.local”) and a value consisting of a JSON array of DNS IPs. | The target nameserver may itself be a Kubernetes service. For instance, you can run your own copy of dnsmasq to export custom DNS names into the ClusterDNS namespace. |
 | `upstreamNameservers` (optional) | A JSON array of DNS IPs. | Note: If specified, then the values specified replace the nameservers taken by default from the node’s `/etc/resolv.conf`. Limits: a maximum of three upstream nameservers can be specified. |
 
-## Additional examples
+## Examples
```

Comment:
In fact, I think both the "Examples" section and the "ConfigMap options" section above should be under "Configure stub-domain and upstream DNS servers".